### DISCUSSION TITLE
AlDente 1.25

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1160] [General]
### Pro

- Added a **Sensor Data widget** to view battery-related data at a glance.

- Improved popover design by adding **transparency effects** to the widgets.

- Added **hysteresis for the heat protection feature**. There is now a minimum duration of 5 minutes for charging and pausing within heat protection. Read more about the updated Heat Protection feature on our blog: [Feature Explanation - Heat Protection](https://apphousekitchen.com/feature-explanation-heat-protection/)

- Temperatures are now **alternatively displayed in Fahrenheit** depending on the macOS preference.

- Calibration mode can now be continued after restarting AlDente or your MacBook.

- Added the option to remember the decision if Calibration Mode should be resumed after an interruption.

- The calibration widget now includes a button to stop the calibration.

- Improved the accuracy of the Power Flow data for MagSafe 2 MacBook Pros.

### All

- AlDente now asks if you want to quit the app when cmd+q is pressed.

- Overhauled AlDente's init and exit systems to further improve stability.

- Updated Dutch translations.

- Fixed a bug that caused the app to hang after discharging in clamshell mode.

- Fixed a bug that caused the charge limit to remain active even when the app was closed on Intel MacBooks.

With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.25'>AlDente 1.25</a>.</em>

**Comment by DamonHD** [comment]
Update Error!  An error occured while extracting the archive.  Please try again later.

(Apple MBA M1, 14.3.1.)

**Reply by MatthiasKerbl** [reply]
Hi @DamonHD,

Thanks for letting us know, we are trying to fix it right now. In the meantime, you can still uninstall the old version and download and install the new version.

Best,
Matthias

**Reply by MatthiasKerbl** [reply]
Hi @DamonHD,

The update error is fixed now.

Best,
Matthias

**Reply by DamonHD** [reply]
Yes, fixed now, thanks!

Damon

**Comment by SoliGit** [comment]
> Update Error! An error occured while extracting the archive. Please try again later.
> 
> (Apple MBA M1, 14.3.1.)

Same.

(Apple MBP M3 Max, 14.2.1)

**Reply by MatthiasKerbl** [reply]
Hi @SoliGit ,

The update error is fixed now.

Best,
Matthias

**Comment by cassperine** [comment]
Thank you for the update!

The "Maximum Capacity" is like "Hardware Battery Percentage" and shows the real health of the battery?
<img width="423" alt="image" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/148076601/9e26ced7-7460-4e47-8962-2fb953f422bc">

**Reply by MatthiasKerbl** [reply]
Hi @cassperine,

We are not yet 100% sure what the different Capacities mean but yes, the Maximum Capacity is the one the battery management system is reporting just like the Hardware Battery Percentage.

Best,
Matthias

**Comment by masterzone3d** [comment]
The sensor data is very good...what is the dfifferences from maximum capacity and macOS capacity?

<img width="392" alt="Screenshot 2024-02-11 at 16 21 15" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/132667740/6a92b838-ffdd-4ca9-a7c0-9e280fb10b18">
